iscript/src/iscript/mac.py

```diff
@@ -1149,6 +1168,10 @@ def get_notarization_status_from_log(log_path):
     entitlements_path = await download_entitlements_file(config, key_config, task)
 
     all_paths = get_app_paths(config, task)
+    langpack_apps = filter_apps(all_paths, fmt="autograph_langpack")
+    if langpack_apps:
```

Comment:
If `sign_langpacks` handles an empty list gracefully, then you don't need this `if` block (we can execute the next 2 lines of code unconditionally). This makes testing coverage easier.

Reply:
I updated testing coverage, and was of the mindset that if we call the "sign langpack" code we should expect to have (at least one) langpack to sign with.